app/script/view_model/content/InputBarViewModel.js

```diff
@@ -489,7 +489,9 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
 
   onInputKeyUp(data, keyboardEvent) {
     this.emojiInput.onInputKeyUp(data, keyboardEvent);
-    this.updateSelectionState();
+    if (keyboardEvent.key !== z.util.KeyboardUtil.KEY.ESC) {
```

Comment:
Shouldn't this be handled by the MentionSuggestion component? I can close itself when the `ESC` key is pressed

Reply:
To clarify: the closing is still handled by the MentionSuggestion component. This guard here just makes sure that it isn't immediatly reopened when `esc` is released.